<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="username_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Inscription</string>
   </property>
  </widget>
  <widget class="QLabel" name="username_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
